src/codeflare_sdk/cluster/cluster.py

```diff
@@ -145,7 +152,7 @@ def down(self):
                 version="v1beta1",
                 namespace=namespace,
                 plural="appwrappers",
-                name=self.app_wrapper_name,
+                name=self.config.name,
```

Comment:
Are `self.app_wrapper_name` and `self.config.name` always the same? Is it safe to make this substitution?

Reply:
Good catch @MichaelClifford, the user could leave the name blank. Added a fix for this.